next.config.js

```diff
@@ -9,6 +9,13 @@ const withTM = require('next-transpile-modules')([
 
 module.exports = withTM({
   webpack: (config, { isServer }) => {
+    config.resolve = {
+      ...config.resolve,
+      alias: {
+        ...config.resolve.alias,
+        '@project-serum/anchor$': '@project-serum/anchor/dist/esm',
```

Comment:
This was only necessary to hotfix broken version of `@project-serum/anchor`, it can be removed now & `@project-serum/anchor` put back to a stable, recent version